bin/run-tests-in-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+# Example:
+# ./bin/run-tests-in-docker.sh
+
```

Comment:
I would suggest adding `set -e` here to ensure that any failure to build the image will cause the script to fail.

```suggestion

# Stop executing when a command returns a non-zero return code
set -e
```